_posts/en/newsletters/2021-09-29-newsletter.md

```diff
@@ -0,0 +1,162 @@
+  version number.
+
+- [Is there historical data of UTXOs by address type?]({{bse}}109776)
+  Murch provides a few example charts from [transactionfee.info][].
+
+- [How are OP_CSV and OP_CLTV backwards compatible?]({{bse}}109834)
+  Andrew Chow describes the historical [soft fork activation][topic soft fork
+  activation] mechanisms of both the BIP65 and BIP112 [timelocks][topic timelocks].
+
+## Preparing for taproot #15: signmessage protocol still needed
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/14-signmessage.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.2rc2][bitcoin core 0.21.2] is a release candidate
+  for a maintenance version of Bitcoin Core.  It contains several bug
+  fixes and small improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #12677][] adds `ancestorcount`, `ancestorsize` and
+  `ancestorfees` fields to the transaction outputs returned by the wallet's
+  `listunspent` RPC method. If the transaction creating the transaction output is
+  unconfirmed, those fields will indicate the total count, size and fees
+  of the transaction and all its unconfirmed ancestors in the mempool.
+  Miners select transactions for inclusion in a block based on their ancestor
+  feerate, so knowing the ancestor size and fees are useful for users estimating
+  the confirmation time for the transaction or attempting to bump the transaction's
+  fee using [CPFP][topic cpfp] or [RBF][topic rbf].
+
+- [Eclair #1942][] enables the path finding algorithm to be configured such that
+  routes with past failures are penalized. This configuration can be applied as
+  an [experimental parameter set][news166 experiments] to potentially improve
+  routing success rates.
+
+- [LND #5101][] adds a *middleware interceptor* which receives each RPC
+  request on its way to the server and can make modifications.  This
+  allows implementing logic outside of LND that can track or affect a
+  large variety of user and automated actions.  For security, only RPCs
+  that explicitly use authentication tokens (macaroons) that opt-in to
+  interception can be intercepted.
+
+- [BTCPay Server #2842][] adds the ability to disable (and later
```

Comment:
I think we could drop this commit from the newsletter. I do not feel strongly about it, however.

Reply:
Yeah, I was on the edge about this, thanks for weighing in!